css-canvas-medical/styles.css

```diff
@@ -0,0 +1,197 @@
+#privacy {
+  margin-right: 40px;
+}
+
+#end {
+  padding: 100px 10%;
+}
+
+.indented {
+  padding-left: 10%;
+  padding-right: 10%;
+}
+
+.relative {
+  position: relative;
+}
+
+.absolute {
+  position: absolute;
+}
+
+.border-bottom {
+  border-bottom: 1px solid rgb(231, 233, 236);
+}
+
+a {
+  text-decoration: none;
+  color: inherit;
+}
+
+a:hover {
+  text-decoration: underline;
+}
+
+body {
+  font-family: Avenir, sans-serif;
+}
+
+h2 {
+  font-family: 'Tiempos Headline', serif;
+}
+
+h3 {
+  font-family: 'Tiempos Headline', serif;
+  font-weight: 300;
+  margin: 0;
+}
+
+.blue-button {
+  background-color: hsl(213deg, 42%, 28%);
+  padding: 20px 50px;
+  border-radius: 4px;
+}
+
+.blue-button:hover {
+  background-color: hsl(213deg, 42%, 18%);
+  text-decoration: none;
+}
+
+.s-three .blue-button {
+  display: block;
+}
+
+.header-link-2 {
+  padding: 15px 17px;
+}
+
+.s-one {
+  margin-top: 35px;
+  margin-bottom: 35px;
+}
+
+.s-two,
+.final {
+  text-align: center;
+  background-color: rgb(250, 250, 250);
+}
+
+footer > .row {
+  padding: 200px 10%;
+}
+
+footer {
+  background-color: rgb(82, 83, 81);
+}
+
+header > .row,
+.s-three > .row {
+  justify-content: space-between;
+}
+
+.s-two .row {
+  justify-content: center;
+}
+
+header > .row > .column-one-half > .row {
+  justify-content: space-between;
+  align-items: center;
+  color: rgb(160, 160, 160);
+}
+
+.s-two .column-one-half {
+  margin-top: 60px;
+  margin-bottom: 60px;
+}
+
+.s-three {
+  margin-top: 40px;
+  margin-bottom: 40px;
+}
+
+.nine-sixteenths-aspect-ratio {
+  width: 100%;
+  padding-top: 56.25%;
+}
+
+.hero {
+  z-index: -1;
+  top: 0;
+  bottom: 0;
+  left: 0;
+  right: 0;
+  background-image: url('./images/hero.jpg');
+  background-size: cover;
+}
+
+.box {
+  top: 0;
+  bottom: 0;
+  left: 6.25%;
+  right: 42.5%;
+}
+
+.box > h2 {
+  margin-top: 60px;
+}
+
+.box > p {
+  margin-top: 60px;
+  margin-bottom: 60px;
+}
+
+.logo {
+  width: 135px;
+}
+
+div.column-one-half {
+  flex-basis: 50%;
+}
+
+div.column-one-fourth {
+  flex-basis: 25%;
+}
+
+div.row {
+  display: flex;
+}
```

Comment:
I wouldn't specify this as `div`s with a `class` of column-one-half, column-one-fourth, or row. What if you want to use the `row` or `column-one-half` classes on elements other than `div`s?